source/clear-linux/tutorials/kata.rst

```diff
@@ -0,0 +1,81 @@
+  .. _kata:
+
+Install Kata Containers
```

Comment:
Suggest changing to:  Kata Containers\*
(OpenSource has submitted for the trademark on TESS)

Reply:
Changed. Thanks